tests/util/hacker.test.util.js

```diff
@@ -391,7 +391,46 @@ const invalidHacker1 = {
         accommodation: {
             dietaryRestrictions: ["Gluten-Free"],
             shirtSize: "L",
-            travel: 0
+            travel: "sfg"
+        }
+    }
+};
+
+// duplicate of newHack0, but with 101 for travel
+const invalidHacker2 = {
```

Comment:
are we missing tests for `impairments` and `barriers`? I'll make a ticket if we are

Reply:
Yes, there are no tests for those currently.